2023/Mofor_Emmanuel-15_05_2023-31_08_2023/week3.md

```diff
@@ -0,0 +1,47 @@
+## Week 3: May 29 - June 2
+
+### What I worked on
+
+My third week at Afkanerd Technologies was heavily focused on researching about Web Application Security, in preparation for my first blog post. I accomplished the following:
+
+<!-- - Attended the weekly standup meeting and got feedback from my previous week's work. -->
+- I did some research on Web Application Security:
+  - Read blog posts on the related topic
+  - Read 2 books on WebAppSec, targeting the vulnerabilities to be addressed, and countermeasures to prevent them.
+
+  (I drew inspiration from these)
+
+- I published a blog post on [Web Application Security](https://moforemmanuel.github.io/blog/web-application-security), addressing the following vulnerabilities:
+
+  - Cross-Site Scripting
+  - SQL Injection
+  - Insecure Session Management
+  - Cross-Site Forgery Request
+
+  I also went ahead to provide practical advice on secure coding techniques, input validation, secure session handling and best practices.
+
+- Sought out areas to improve in my blogging environment.
+
+### What I learned
+
+Through completing these tasks, I learned the following:
+
+- An Overview of Web Application Security
+- How to prevent XSS, SQLi, and CSFR attacks
+- How to write secure code
+
+### Challenges I faced
+
+- **Starting and structuring my first blog post**: As a newbie in the ecosystem of technical writing and a wannabe skilled writer, it truly is an art to craft a beautiful piece that captures the undivided attention of the reader, and I want to master that art. Starting out my first blog was very bewildering, cause I didn't want to sound boring, neither did I want to sound too nerdy, I needed that balance. Another challenge was content structuring, what should come when, where and why?
+
+### How I overcame them
+
+- Pertaining to structuring, with the help of my mentor (a master artist in the ecosystem of technical writing by the way), I was able to follow a structure he provided, and also learned how to structure content, thereupon making things easier for me and eliminating the diversity in mind, in which I was able to focus, since I knew my target.
+- As for starting out the post, I sought to begin with a captivating story, that makes the reader see how ignoring WebAppSec in developing applications can be detrimental, that way, triggering him to want to know more about this WebAppSec and not repeat errors of the past. I also made sure to have a balanced content, not boring, not too nerdy, spanning across relatable real life scenarios, even a tech-unsavvy person can get what's going on.
```

Comment:
In your writing, please avoid attributing gender identities to your audience. It's important to maintain an inclusive approach since you don't know the gender identity of the person who will be reading the article. By using gender-neutral pronouns or rephrasing sentences, you can ensure that your content appeals to a diverse audience.

Reply:
Alright @PromiseFru, understood. Thanks for the feedback.